Update README.md

Whytehorse opened the pull request:
Added bill of rights to constitution as article 13

Whytehorse committed:
Update README.md

xsttx commented:
@Whytehorse all of these are very different from each other. I suggest you do pull request for one at the time, under relevant articles, otherwise it's nearly impossible to comment on or merge.

Whytehorse commented:
@xstt What? It's one article and it's new

Whytehorse commented:
it's the universal declaration of human rights put into simple english. It's one article called rights. Please pull

Whytehorse committed:
Article 14: centralization prohibited

resolves issue 21 https://github.com/Bit-Nation/BITNATION-Constitution/issues/21

Whytehorse committed:
Merge pull request #2 from Bit-Nation/master

Resync with master

Whytehorse committed:
Create testflightdelay.sol

Whytehorse committed:
Create readme.md

Whytehorse committed:
Merge pull request #1 from Whytehorse/Whytehorse-patch-1

Article 14: centralization prohibited

Whytehorse commented:
I added insurance dApp folder and readme plus sample code

ETeissonniere commented:
@Whytehorse that content doesn't fit the "update readme" title, can you please different pull req for the different part? Thanks.

ETeissonniere commented on dapp/src/insurance/testflightdelay.sol:
```diff
@@ -0,0 +1,745 @@
+	All times are UTC.
+	Copyright (C) Christoph Mussenbrock, Stephan Karpischek
+	
+*/
```
Seems to have been copy pasted from an other repo, better to put a link to that document in dapp/src/insurance/readme.md

ETeissonniere commented on dapp/src/insurance/testflightdelay.sol:
```diff
@@ -0,0 +1,745 @@
+
+
+import "./github.com/oraclize/ethereum-api/oraclizeAPI.sol";
+import "./github.com/Arachnid/solidity-stringutils/strings.sol";
```
The following imports are not valid, they are loading files from a directory which is not included

ETeissonniere replied:
It is also highly recommended not to import code from URLs, those files could be modified by their owner without the end user noticing it

ETeissonniere commented on dapp/src/insurance/testflightdelay.sol:
```diff
@@ -0,0 +1,745 @@
+	// URLs and query strings for oraclize
+
+	string constant oraclize_RatingsBaseUrl =
+		"[URL] json(https://api.flightstats.com/flex/ratings/rest/v1/json/flight/";
```
The whole contract makes use of oraclize to perform external calls, I am sure there is a way to do everything on chain and in a simpler way

Whytehorse reviewed (commented):
test review
